domains/_autodiscover._tcp.ciaobot.json

```diff
@@ -0,0 +1,9 @@
+{
+  "owner": {
+    "username": "Ciao287",
+    "discord": "687333016921440317"
+  },
+  "record": {
+    "SRV": "mail.ciaohost.tech 443"
```

Comment:
i record SRV non so formattati così, vedi https://is-a.dev/docs/domain-structure/